Issue #66: Update documentation along with code changes

Luolc opened the pull request:
#66 

the order of option changed.
localGitRepo => checkstyleLocalRepo
stopAfterGenerateConfig => stopAfterConfigGeneration

rnveach commented:
@Luolc 
> localGitRepo => checkstyleLocalRepo
stopAfterGenerateConfig => stopAfterConfigGeneration

This is 2 issues. Main issue has 4 links.
Will you do more PRs or will this fix everything?

rnveach commented:
@Luolc rebase on latest master and change names in CI.

Luolc commented:
@rnveach 
romani's comments in #59 are all covered.

> Arguments' repoPath should change too to be similar to checkstyleLocalRepo

I changed the related variables to `checkstyleRepoPath` now.

> rebase on latest master and change names in CI if they need it.

Rebased.
